doc/overview/build.md

```diff
@@ -0,0 +1,127 @@
+
+### Freedesktop SDK
+
+Freedesktop SDK provides integration instructions for a Linux runtime,
+using components commonly used in many Linux distributions.
+
+Some documentation for Freedesktop SDK is available at:
+<https://freedesktop-sdk.gitlab.io/documentation/>.
+
+Freedesktop SDK defines elements for many open source components in the
+[components/ subdirectory](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/tree/master/elements/components).
+These roughly map to packages available in package-based Linux distros.
+
+Higher level groupings are available as stacks in the
+[public-stacks/ subdirectory](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/tree/master/elements/public-stacks?ref_type=heads).
+These can be useful as dependencies when defining your own elements.
+
+### GNOME Build Metadata
+
+The GNOME project maintains build instructions for the entire GNOME platform
+in the repo
+[gnome-build-meta.git](https://gitlab.gnome.org/GNOME/gnome-build-meta/).
+
+This repo is a BuildStream project which uses Freedesktop SDK to provide the
+base platform (via a junction in
+[freedesktop-sdk.bst](https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/master/elements/freedesktop-sdk.bst)).
+
+GNOME OS is also defined in gnome-build-meta.git, with most of the integration
+instructions in the
+[gnomeos/ subdirectory](https://gitlab.gnome.org/GNOME/gnome-build-meta/-/tree/master/elements/gnomeos). Note that GNOME OS defines the image build process using BuildStream,
+unlike Endless OS which uses eos-image-builder for that.
+
+### Endless OS Build Metadata
+
+Now we get to [eos-build-meta.git](https://github.com/endlessm/eos-build-meta)
+where the magic happens.
+
+The base platform for EOS7 is Freedesktop SDK, included via the
+[freedesktop-sdk.bst junction element](https://github.com/endlessm/eos-build-meta/blob/main/elements/freedesktop-sdk.bst). This provides build tools and many system components.
+In eos-build-meta, the junctioned elements are referred to via the junction, e.g.
+`components/podman.bst` becomes `freedesktop-sdk.bst:components/podman.bst`.
+
+The integration instructions for some Freedesktop SDK elements are overridden to
+alter how they are built or configured for EOS. Some of these overrides have to
+be copied manually from GNOME OS.
+
+All of GNOME is brought in via the
+[gnome-build-meta.bst junction element](https://github.com/endlessm/eos-build-meta/blob/main/elements/gnome-build-meta.bst).
+There are some overrides here too. In particular, the nested junction element
+`gnome-build-meta.bst:freedesktop-sdk.bst` is overriden with our own local
+junction element.
+
+The eos-build-meta project defines integration instructions for more Endless specific
+components.
+
+It also defines the final filesystem tree for EOS7, which is done in the following
+elements:
+
+  * [`eos/deps.bst`](https://github.com/endlessm/eos-build-meta/blob/main/elements/eos/deps.bst),
```

Comment:
Is this a way lists are handled in MarkDown? /gen I'm not used to seeing them not start on the first row.

Reply:
I think I have been brain-damaged by JIRA's markdown syntax, but it does work